types/chart.js/index.d.ts

```diff
@@ -239,6 +240,7 @@ declare namespace Chart {
         animation?: ChartAnimationOptions;
         elements?: ChartElementsOptions;
         layout?: ChartLayoutOptions;
+        scale?: ChartScales;
```

Comment:
`scale` and `scales`, which is correct? Can the either of them be removed?

Reply:
Both are used, ´scale´ is used in the radio chart while `scales` is used, for example, in the bar chart.

Reply:
It is true, both ```scale``` and ```scales``` exist, but I think they don't implements the same interface. According to the documentation you provided, there is only one property:

```
options = {
    scale: {
        // Hides the scale
        display: false
    }
};
```

Reply:
You are right, now I updated it.